Dockerfile

```diff
@@ -16,7 +16,7 @@ ADD . /chronos
 
 WORKDIR /chronos
 
-RUN mvn clean package
```

Comment:
My "docker build" run was successful. It seems the tests are flaky, but I think they should  be fixed rather than skipped.

So I'd remove this change from the PR.